src/uct/api/uct.h

```diff
@@ -1432,23 +1448,52 @@ struct uct_listener_params {
  */
 struct uct_md_attr {
     struct {
-        uint64_t             max_alloc; /**< Maximal allocation size */
-        size_t               max_reg;   /**< Maximal registration size */
-        uint64_t             flags;     /**< UCT_MD_FLAG_xx */
-        uint64_t             reg_mem_types; /**< Bitmap of memory types that Memory Domain can be registered with */
-        uint64_t             cache_mem_types; /**< Bitmap of memory types that can be cached for this Memory Domain */
-        uint64_t             detect_mem_types; /**< Bitmap of memory types that Memory Domain can detect if address belongs to it */
-        uint64_t             alloc_mem_types;  /**< Bitmap of memory types that Memory Domain can allocate memory on */
-        uint64_t             access_mem_types; /**< Memory types that Memory Domain can access */
+        /** Maximal allocation size */
+        uint64_t max_alloc;
+
+        /** Maximal registration size */
+        size_t   max_reg;
+
+        /** UCT_MD_FLAG_xx */
+        uint64_t flags;
+
+        /** Bitmap of memory types that Memory Domain can be registered with */
+        uint64_t reg_mem_types;
+
+        /**< Bitmap of memory types that can be cached for this Memory Domain */
+        uint64_t cache_mem_types;
+
+        /** Bitmap of memory types that Memory Domain can detect if address
+         *  belongs to it */
+        uint64_t detect_mem_types;
+
+        /** Bitmap of memory types that Memory Domain can allocate memory on */
+        uint64_t alloc_mem_types;
+
+        /** Memory types that Memory Domain can access */
+        uint64_t access_mem_types;
```

Comment:
unrelated

Reply:
we decided to move the comments before the field
https://github.com/openucx/ucx/pull/8422#discussion_r939989576